performance/SwiftUIPerformanceQuickstart/PerformanceExample/Podfile

```
@@ -2,6 +2,7 @@ use_frameworks!
 
 pod 'Firebase'
 pod 'FirebasePerformance'
+pod 'FirebaseStorage'
```

```suggestion
pod 'FirebaseStorage'
pod 'FirebaseStorageSwift'
```
I imagine that as a follow-up PR, we could try to use `FirebaseStorageSwift` which would enable Storage async/await functionality: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseStorageSwift/CHANGELOG.md.

Currently Storage upload functionality relies on completion handlers even in async/await contexts.

---

Done ✅